app/src/components/multistepform.js

```diff
@@ -19,7 +19,7 @@ export default function MultiStepForm({children, submitText, handleSubmit}) {
           Children.map(children, (child, idx) => {
             if (idx === step) {
              return (
-                <div className="block font-bold pl-5 pt-3 pb-3 bg-blue-200 rounded-lg">
+              <div className="block font-bold pl-5 pt-3 pb-3 rounded-lg" style={{backgroundImage: 'linear-gradient(to bottom, #D799D5, #B5B9F1)'}}>
```

Comment:
Apparently tailwind does not support top down gradients which is why I had to use regular css here

Reply:
Yep - but this looks good! :)